src/diffpy/srmise/dataclusters.py

```diff
@@ -82,12 +82,28 @@ def __eq__(self, other):
         )
 
     def clear(self):
-        """Remove all the points from the cluster list,
-        and reset x, y value sequences to empty list. Reset the resolution to 0."""
+        """
+        Clear all data and reset the cluster object to a transient initial state.
+
+        This method performs the following steps:
```

Comment:
this block is too much information, so we can remove it.  It is information that is probably not needed by a user and a developer can look at the code to get the precise definition.